### PR TITLE
Finishing up the placeholders in FAST_Solver.f90 for ExtLoads module

### PR DESCRIPTION
**Pull request by ashesh2512**
Also updated the CmakeLists to the latest corresponding to opencast/dev. Involves minor changes when installing tests and other executables.

### CODE REVIEW

File: modules-local/fast-library/src/FAST_Solver.f90 [hunk 1]
```diff
@@ -4081,20 +4146,73 @@ SUBROUTINE InitModuleMappings(p_FAST, ED, BD, AD14, AD, ExtLd, HD, SD, ExtPtfm,
...
+            CALL MeshMapCreate( ExtLd%y%BladeLoad(K), ED%Input(1)%BladePtLoads(K),  MeshMapData%ExtLd_P_2_BDED_B(K), ErrStat2, ErrMsg2 )
+            CALL SetErrStat( ErrStat2, ErrMsg2, ErrStat, ErrMsg, RoutineName//':ExtLd_P_2_BDED_B('//TRIM(Num2LStr(K))//')' )
+         END DO
+
+      ELSEIF ( p_FAST%CompElast == Module_BD ) then
+
+         ! connect ExtLoads mesh with BeamDyn
+         DO K=1,NumBl
+            ! Create mapping for BeamDyn BldMotion line2 meshes to ExtLoads point mesh
+            CALL MeshMapCreate( BD%y(k)%BldMotion, AD%Input(1)%BladeMotion(K), MeshMapData%BDED_L_2_AD_L_B(K), ErrStat2, ErrMsg2 )
```
Comment (gantech) [inline modules-local/fast-library/src/FAST_Solver.f90]: Does this map to AeroDyn?

Comment (gantech) [reply modules-local/fast-library/src/FAST_Solver.f90]: I guess the same question holds for the maps around this line.

File: modules-local/fast-library/src/FAST_Solver.f90 [hunk 2]
```diff
@@ -214,7 +216,11 @@ SUBROUTINE ED_InputSolve( p_FAST, u_ED, y_ED, p_AD14, y_AD14, y_AD, y_SrvD, u_AD
 
       ELSE IF (p_FAST%CompAero == Module_ExtLd ) THEN
 
-         ! DO SOMETHING HERE
+         DO K = 1,SIZE(u_ED%BladePtLoads,1) ! Loop through all blades (p_ED%NumBl)
+           ! NOTE - not only is BladeLn2Mesh not a Sbiling of TowerPtLoads, it is a line 2 mesh with different number of nodes
```
Comment (gantech) [inline modules-local/fast-library/src/FAST_Solver.f90]: I don't understand this comment. I'd expect blade meshes to be completely different from tower meshes.

Comment (gantech) [reply modules-local/fast-library/src/FAST_Solver.f90]: Ah.. Seeing the similar comment for the tower you made below tells me this is probably a typo. Are y_ED%BladeLn2Mesh and u_ED%BladePtLoads sibling meshes. If not, we probably have to understand this better.

File: modules-local/fast-library/src/FAST_Solver.f90 [hunk 3]
```diff
@@ -258,7 +264,10 @@ SUBROUTINE ED_InputSolve( p_FAST, u_ED, y_ED, p_AD14, y_AD14, y_AD, y_SrvD, u_AD
 
    ELSE IF (p_FAST%CompAero == Module_ExtLd ) THEN
 
-         ! DO SOMETHING HERE
+      IF ( y_ExtLd%TowerLoad%Committed ) THEN ! NOTE - not only is TowerLn2Mesh not a Sbiling of TowerPtLoads, it is a line 2 mesh with different number of nodes
```
Comment (gantech) [inline modules-local/fast-library/src/FAST_Solver.f90:267]: Hmm..Need to understand this better.

File: modules-local/fast-library/src/FAST_Solver.f90 (hunk 1 above)
Comment (ashesh2512) [reply modules-local/fast-library/src/FAST_Solver.f90]: This has been fixed. It now maps to External loads module

File: modules-local/fast-library/src/FAST_Solver.f90 (hunk 3 above)
Comment (ashesh2512) [reply modules-local/fast-library/src/FAST_Solver.f90:267]: This and the map above, both use displacement meshes that aren't siblings of the load meshes.  Not only that, but they both are created using different number of nodes.

**Comment by gantech** [comment]
Nice. Thanks @ashesh2512

**Pull request merged**